core/storage/question/gae_models.py

```diff
@@ -192,21 +194,37 @@ def create(cls, question_id, skill_id):
         return question_skill_link_model_instance
 
     @classmethod
-    def get_question_ids_linked_to_skill_ids(cls, skill_ids):
+    def get_question_ids_linked_to_skill_ids(cls, skill_ids, start_cursor):
         """Creates a new QuestionSkillLinkModel entry.
 
         Args:
             skill_ids: list(str). The ids of skills for which the linked
                 question ids are to be retrieved.
+            start_cursor: str. The starting point from which the batch of
+                questions are to be returned.
 
         Returns:
-            list(str). The question ids linked to given skills.
+            list(str), str. The question ids linked to given skills and the next
+                cursor value to be used for the next page.
         """
-        question_skill_link_models = cls.query(cls.skill_id.IN(skill_ids))
+        if not start_cursor == '':
+            cursor = datastore_query.Cursor(urlsafe=start_cursor)
+            question_skill_link_models, next_cursor, more = cls.query( #pylint: disable=unused-variable
```

Comment:
I think you should pass "more" back. That way the frontend will know "is there another page coming?"

Also, note: if you don't actually use a variable, replace it with `_`. That way pylint shouldn't trigger.

Reply:
Actually, one of the backend tests failed due to this as next_cursor.urlsafe() doesn't work if next_cursor is None, which happens if there aren't any more elements. 
So, shall I use this check to set next_cursor as a blank string if no more pages are left, in which case more is not needed?

Reply:
But that's why you pass "more" back. So that the frontend doesn't send None as the next cursor.

Reply:
Oh, I see, you're thinking of using the next_cursor as a signal here. I am not certain whether the next_cursor can be non-None even if there are no more queries, you'll need to check the App Engine documentation for that.

I'm happy with any way that works. I know we use "more" in search_services and elsewhere though, so might want to just go with that for consistency.